TRAMS-API/Mapping/GetProjectAcademiesResponseMapper.cs

```diff
@@ -0,0 +1,60 @@
+﻿using API.Models.Downstream.D365;
+using API.Models.Upstream.Enums;
+using API.Models.Upstream.Response;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.Mapping
+{
+    public class GetProjectAcademiesResponseMapper : IMapper<AcademyTransfersProjectAcademy,
+                                                             Models.Upstream.Response.GetProjectsAcademyResponseModel>
+    {
+        public GetProjectsAcademyResponseModel Map(AcademyTransfersProjectAcademy academy)
+        {
```

Comment:
Suggestion: On other mappers you check if the input is null before performing an action.  Could be useful here to guard against nulls also.

Reply:
Will fix